docs/source/quickstart.rst

```diff
@@ -1,81 +1,187 @@
 
 
-2. Download the data
+2. Download the data 💾
 -----------------------------------
 
-2.1 Simulated data sets for all four instruments are available for the Transiting Exoplanet Community ERS Data Challenge No. 1.  Download a simulated data set of your choice from the `STScI Box site <https://stsci.app.box.com/s/tj1jnivn9ekiyhecl5up7mkg8xrd1htl/folder/154382715453>`_.
+With the installation complete, you'll need some data to run ``Eureka!`` on. For now let's use some simulated data that was produced for the Transiting Exoplanet Community ERS Data Challenge. Datasets for all four instruments are available on the `STScI Box site <https://stsci.app.box.com/s/tj1jnivn9ekiyhecl5up7mkg8xrd1htl/folder/154382715453>`_, however, for the rest of this quickstart guide the `NIRSpec Tiny dataset <https://stsci.box.com/s/mgicm6yc5c7khljako7yswh619dn5e7a>`_ will be used. 
 
-2.2. Make a new directory to store the data.  For example, if you want to work with NIRSpec data, you could structure your directories like this:
+Now, I'm sure you wouldn't just leave the data in your Downloads folder, but if so, let's make a new directory to store things instead. For example:
 
 .. code-block:: bash
 
 	mkdir /User/Data/JWST-Sim/NIRSpec/
 	cd /User/Data/JWST-Sim/NIRSpec/
-	mkdir Stage2
+	unzip -j ~/Downloads/Tiny.zip -d .
+
+Note that for Eureka! you do *not* need to download any ancillary data - any additional files will be downloaded automatically (if you correctly set the CRDS environment variables during installation). 
+
+
+3. Set up your run directory 🗂
+-------------------------------
+
+3.1 Gather the demo files
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-Note that you do *not* need to download any ancillary data --- any additional files will be downloaded automatically (remember to set the CRDS environment variables when you install ``Eureka!``). 
+We're almost there, but before you can get things running you need to set up a directory for ``Eureka!`` to store both input and output files. 
 
+.. code-block:: bash
+	
+	mkdir /User/DataAnalysis/JWST/MyFirstEureka
+	cd /User/DataAnalysis/JWST/MyFirstEureka
+
+From here, the simplest way to set up all of the Eureka input files is to duplicate them from the JWST demos directory within ``Eureka!``. This can be done using your existing installation:
 
-3. Set up your run directory 
------------------------------------------------------------------
+.. code-block:: bash
+	
+	mkdir demos
+	cp -r User/Eureka/Install/Location/demos/JWST/* ./demos
 
-3.1   Copy the JWST demos directory from your ``Eureka!`` installation to wherever you want to run your analysis. For example,
+Or, if you're lost in the depths of your conda directory, you can also download the demos folder directly `here <https://downgit.github.io/#/home?url=https://github.com/kevin218/Eureka/tree/main/demos/JWST>`_ and copy them over:
```

Comment:
The demo files don't end up getting packaged with the pip installation, so they aren't in the conda directory.

Also, if one doesn't know where to find the folder for a package they can do `pip show eureka` and it'll specify where the package was installed

Reply:
Wait, this downgit link you made is super cool!!! That's fantastic to have!

Reply:
The `pip show eureka` is equally useful to me! For future reference, you can get a link like this using https://downgit.github.io

All these changes look good, I'll take a stab after the meeting on Monday. Maybe if i change to the lsq I can fit more than two transits haha.